crates/common/src/config.rs

```diff
@@ -6,6 +6,39 @@ use citrea_pruning::PruningConfig;
 use serde::de::DeserializeOwned;
 use serde::{Deserialize, Serialize};
 use sov_stf_runner::ProverGuestRunConfig;
+
+pub trait FromEnv: Sized {
+    fn from_env() -> anyhow::Result<Self>;
+}
+impl FromEnv for PruningConfig {
+    fn from_env() -> anyhow::Result<Self> {
+        Ok(PruningConfig {
+            distance: std::env::var("PRUNING_DISTANCE")?.parse()?,
+        })
+    }
+}
+impl FromEnv for sov_mock_da::MockDaConfig {
+    fn from_env() -> anyhow::Result<Self> {
+        Ok(Self {
+            sender_address: std::env::var("SENDER_ADDRESS")?.parse()?,
+            db_path: std::env::var("DB_PATH")?.into(),
+        })
+    }
+}
```

Comment:
should be implemented in mock da

Reply:
kept this version since implementing it that way causes cyclic dependency issues in the tests